docs/modules/ROOT/pages/explanations/backup.adoc

```diff
@@ -0,0 +1,62 @@
+= K8up Backup Methods
+
+K8up currently implements three ways to do backups:
+
+. xref:how-tos/backup.adoc[_PVC backups_], which work by mounting the underlying `PV` of each `PVC` and reading its files.
+. xref:how-tos/application-aware-backups.adoc[_Backup commands_], which print the content, that should be backed up, to STDOUT.
+. The special xref:how-tos/prebackuppod.adoc[`PreBackupPod`], which is run as part of a backup.
+
+This explanation guide briefly explains each backup method.
+
+== PVC Backup
+
+This is the most straight-forward backup approach.
+What k8up basically does is that it looks at all the `PVC` definitions in a namespace.
+For every `PVC` it creates a `Job` in that same namespace.
+That `Job` will launch a `Pod`, which mounts the respective `PV`.
+All the content of that `PV` is then backed-up.
+
+This method's advantage is that it is dead-easy to use and works for a surprising large amount of use-cases.
+It does not work in some cases though.
+More precise, it does not work when files are kept open for a long period of time, like databases do.
+It does also not work for content that is not stored in the cluster, like a managed database that is offered by a service provider.
+
+== Application-Aware Backups
```

Comment:
Maybe we could add a note that tar and stdout backups are a method to circumvent the RWO problem for the time being? I know of a few customer that do it that way.

E.g. backupcommand of `tar -czf - /data`. But not sure if this is something for an advanced use cases section or so.